site/docs/main/customize-installation.md

```diff
@@ -95,10 +97,108 @@ velero client config set features=
 
 ## Customize resource requests and limits
 
-By default, the Velero deployment requests 500m CPU, 128Mi memory and sets a limit of 1000m CPU, 256Mi.
-Default requests and limits are not set for the restic pods as CPU/Memory usage can depend heavily on the size of volumes being backed up.
+Velero sets resource requests and limits by defaults at installation for the Velero pod and the restic pod, if you using the [restic integration](/docs/main/restic/).
 
-Customization of these resource requests and limits may be performed using the [velero install][6] CLI command.
+<table caption="Velero Customize resource requests and limits defaults" >
+  <tr><th>Setting</th><th>Velero pod defaults</th><th>restic pod defaults (Velero 1.4.2 and later)</th></tr>
+  <tr><td>CPU request</td><td>500m</td><td>500m</td></tr>
+  <tr><td>Memory requests</td><td>128Mi</td><td>512Mi</td></tr>
+  <tr><td>CPU limit</td><td>1000m (1)</td><td>1000m (1)</td></tr>
+  <tr><td>Memory limit</td><td>256Mi</td><td>1024Mi</td></tr>
+</table>
+
+### Install with custom resource requests and limits
+
+You can customize these resource requests and limit when you first install using the [velero install][6] CLI command.
```

Comment:
For other reviewers: I tested updating pod requests/limits with `velero install`, and it does not work. `kubectl edit` is required.